doc/authorization.md

```diff
@@ -0,0 +1,165 @@
+Due to a limitation in the LXD client, if `can_exec` is granted to a group for a particular instance, members of the group will not be able to start a terminal session unless `can_view_events` is additionally granted for the parent project of the instance.
+We are working to resolve this.
+```
+
+(identities)=
+### Explore identities
+
+To discover available identities that may be assigned to a group, or view identities that are currently assigned, run:
+
+    lxc auth identity list
+
+The authentication method column displays the method by which the client authenticates with LXD.
+
+The type column displays the type of identity.
+Identity types are a superset of TLS certificate types and additionally include OIDC clients.
+
+The name column displays the name of the identity.
+For TLS clients, this will be the name of the certificate as shown in the certificates API.
+For OIDC clients this will be the name of the client as given by the IdP (requested via the [profile scope](https://openid.net/specs/openid-connect-basic-1_0.html#Scopes)).
+
+The identifier column displays a unique identifier for the identity within that authentication method.
+For TLS clients, this will be the certificate fingerprint.
+For OIDC clients, this will be the email address of the client.
+
+The groups column displays any groups that are currently assigned to the identity.
+Groups cannot currently be assigned to TLS clients.
+
+```{note}
+For OIDC clients to be displayed in the list of identities, they must first add the LXD server as a remote.
+```
+
+(access-management)=
+### Access management
+
+In LXD, identities cannot be granted permissions directly. Instead, permissions are added to groups, and groups are added to identities.
+To create a group, run:
+
+    lxc auth group create <group_name>
+
+To add a group to an identity, run:
+
+    lxc auth identity group add <authentication_method>/<identifier> <group_name>
+
+For OIDC clients, this is:
+
+    lxc auth identity group add oidc/<email_address> <group_name>
+
+The identity is now a member of the group. To add permissions to the group, run:
+
+    lxc auth group permission add <group_name> <entity_type> [<entity_name>] <entitlement> [<key>=<value>...]
+
+Here are some examples:
+
+- `lxc auth group permission add administrator server admin` grants members of `administrator` the `admin` entitlement on `server`.
+- `lxc auth group permission add junior-dev project sandbox operator` grants members of `junior-dev` the `operator` entitlement on project `sandbox`.
+- `lxc auth group permission add my-group instance c1 user project=default` grants members of `my-group` the `user` entitlement on instance `c1` in project `default`.
+
+Some entity types require more than one supplementary argument to uniquely specify the entity.
+For example, entities of type `storage_volume` and `storage_bucket` require an additional `pool=<storage_pool_name>` argument.
+
+(identity-provider-groups)=
+### Identity provider groups
```

Comment:
```suggestion
### Use groups defined by the identity provider
```

(I'm not quite happy with "use" here, but I can't think of anything better right now ...)